components/01-atoms/disclosure/_disclosure.scss

```diff
@@ -0,0 +1,100 @@
+.disclosure {
+  position: relative;
+
+  &[data-component-layout='stacked'] {
```

Comment:
I think this is a great component to have in here, but I'm concerned it's doing more than a "starter" component should do.

In my experience, when a site has an accordion, there's only one style (e.g. stacked or tooltip) and one alignment (usually left, maybe centered... But I don't think I've ever seen one right aligned), and the same with the position.

So... I think that all of the options are nice, and if we were building a framework where the off-the-shelf components should work for any use-case, this would be the way to go. However, that's not the point of the Compound components. They're intended to be stripped-down, basic components that we (at 4K, or anyone else that wants a starting point) can pull into a project, and then tweak _for that specific project_.

Something we've talked about doing, that I think this code would be good for is creating a "highly designed" component set - separate from Compound - that _could_ be used off-the-shelf, or as a way of demonstrating what Emulsify can do, out of the box.

I think that would be a great place for this code, as-is. We haven't created that repo yet... I'm thinking about calling it "Synthesis".

Let's talk over this when you're back from vacation.